### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (467)
Screenshot (468)
Screenshot (469)
=

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+@import url(../../root.css);
+@import url('https://pro.fontawesome.com/releases/v5.10.0/css/all.css');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap');
+html{
+    font-family: 'Montserrat', sans-serif;
+}
+
+.grid-head,
+.grid-topic {
+    color: var(--font-color);
```
Comment (hinanshisuthar) [inline components/grid/grid.css:11]: Good use of variables

File: components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+}
+.diff-gr{
+    border-radius: 5px;
+    text-align: center;
+    padding: 10px;
+}
+.gr-item{
+    background-color: var(--primary-color);
+    border-radius: 5px;
+    /* height: 100%; */
```
Comment (hinanshisuthar) [inline components/grid/grid.css:131]: Commented code can be removed

**Review by hinanshisuthar — APPROVED** [review]
- Good use of components wherever possible
- Nicely structured code
- Functionality works as intended
- Nice work done. Good to merge from my side

**Review by ZuberDunge — APPROVED** [review]
just a small suggestion - extra space can be removed.
rest everything looks fine,  good to go from my side.

File: components/grid/grid.css (hunk 1 above)
Comment (Megha-Joshi) [reply components/grid/grid.css:11]: +1

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+.grid-content{
+
+    padding: var(--content-padding);
+    font-size: var(--content-font);
+    color: var(--primary-color);
+}
+
+
```
Comment (Megha-Joshi) [inline components/grid/grid.css:30]: Give proper spacing between two classes.

File: components/grid/grid.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    border-radius: var(--border-radius);
+    justify-content: center;
+    align-items: center;
+}
+
+.grid-box-simple{
+    display: grid;
+    grid-template-columns: repeat(3, minmax(0, 1fr));
+    grid-template-rows: repeat(2, minmax(0, 1fr));
+    width: 500px;
```
Comment (Megha-Joshi) [inline components/grid/grid.css]: As the px is big, change it to rem.

File: components/grid/grid.css [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    gap: 10px;
+
+}
+.grid-box-uneven, .grid-box-col2{
+    display: grid;
+    width: 500px;
```
Comment (Megha-Joshi) [inline components/grid/grid.css]: Change px to rem as the value of px is very big.

File: components/grid/grid.css (hunk 2 above)
Comment (Megha-Joshi) [reply components/grid/grid.css:131]: +1

File: components/grid/grid.css [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+}
+.gr-item{
+    background-color: var(--primary-color);
+    border-radius: 5px;
+    /* height: 100%; */
+    text-align: center;
+    padding: 10px;
+    color: var(--whitesmoke-color);
+
+}
```
Comment (Megha-Joshi) [inline components/grid/grid.css:136]: Give proper spacing.

File: components/grid/grid.css [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+span {
+    font-size: var(--content-font);
+}
+
+
+.code{
+    margin: 1rem auto;
+    width: 400px;
```
Comment (Megha-Joshi) [inline components/grid/grid.css:148]: Change px to rem

Comment (Megha-Joshi) [inline components/grid/grid.css:148]: Change px to rem

File: components/grid/grid.html [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+        <div class="document-container">
+            <div class="sidebar">
+                <div class="doc-topic">
+                    Documentation
+                </div>
+
+                <ul class="doc-list">
+                    <li class="doc-item"><a href="../../docs.html " class="side-link">Introduction</a></li>
+                    <li class="doc-item"><a href="../color/color.html" class="side-link">Colors</a></li>
+
```
Comment (Megha-Joshi) [inline components/grid/grid.html:52]: Remove this extra spacing.

Comment (Megha-Joshi) [inline components/grid/grid.html:52]: Remove this extra spacing.

**Review by Megha-Joshi — APPROVED** [review]
- Great UI. 
- Naming convention is impressive.
- Clean and readable code.
- Just do the suggested changes. Otherwise good to go!!

**Pull request merged**